sympy/solvers/ode.py

```diff
@@ -290,10 +290,10 @@
     "separable_reduced",
     "1st_power_series",
     "lie_group",
-    "nth_linear_constant_coeff_homogeneous",
-    "nth_linear_euler_eq_homogeneous",
     "nth_linear_constant_coeff_undetermined_coefficients",
     "nth_linear_euler_eq_nonhomogeneous_undetermined_coefficients",
+    "nth_linear_constant_coeff_homogeneous",
+    "nth_linear_euler_eq_homogeneous",
```

Comment:
I referred to one of your comments:
https://github.com/sympy/sympy/issues/5334#issuecomment-36999456